lms/djangoapps/course_home_api/progress/v1/views.py

```diff
@@ -127,11 +126,15 @@ def get(self, request, *args, **kwargs):
             reset_masquerade_data=True
         )
 
-        course = get_course_with_access(request.user, 'load', course_key, check_if_enrolled=True)
```

Comment:
Oh wait, was the issue this wasn't properly redirecting despite returning a 302 status? Wild. Or was this the CORS issue?

Reply:
Yeah it wasn't properly redirecting, and if it was working as intended it would have redirected to the course about page (which is not the behavior we want).